ASCOM.Driver/TelescopeDriver/Driver.cs

```diff
@@ -116,26 +116,26 @@ public string Action(string ActionName, string ActionParameters)
 				switch (ActionName)
 				{
 					case "Telescope:getFirmwareVer":
-					{
-						retVal = CommandString(":GVP#,#"); // Get firmware name
-						retVal = retVal + " " + CommandString(":GVN#,#"); // Get firmware version number
-						break;
-					}
+						{
+							retVal = CommandString(":GVP#,#"); // Get firmware name
+							retVal = retVal + " " + CommandString(":GVN#,#"); // Get firmware version number
+							break;
+						}
 
 					case "Utility:JNowtoJ2000":
-					{
-						_transform.SetTopocentric(System.Convert.ToDouble(ActionParameters.Split(',')[0]),
-							System.Convert.ToDouble(ActionParameters.Split(',')[1]));
-						retVal = _utilities.HoursToHMS(_transform.RAJ2000, ":", ":", string.Empty) + "&" +
-									DegreesToDmsWithSign(_transform.DecJ2000, "*", ":", string.Empty);
-						break;
-					}
+						{
+							_transform.SetTopocentric(System.Convert.ToDouble(ActionParameters.Split(',')[0]),
```

Comment:
_oatCulture

Reply:
This does not process OAT commands. I'm not even sure it is used anywhere, but even if it is, it would be coming from an ASCOM call and they don't force a culture.